### PR TITLE
Add slcli account licenses

### PR DESCRIPTION
**Pull request by caberos**
https://github.com/softlayer/softlayer-python/issues/1500

### CODE REVIEW

File: SoftLayer/managers/account.py [hunk 1]
```diff
@@ -303,3 +303,25 @@ def get_network_message_delivery_accounts(self):
         _mask = """vendor,type"""
 
         return self.client['SoftLayer_Account'].getNetworkMessageDeliveryAccounts(mask=_mask)
+
+    def get_active_virtual_licenses(self):
+        """Gets all active virtual licenses account.
+
+                :returns: active virtual licenses account
+                """
```
Comment (allmightyspiff) [inline SoftLayer/managers/account.py:311]: Indentation should be the same as the start of this comment block 
```suggestion
        """Gets all active virtual licenses account.

        :returns: active virtual licenses account
        """
```

File: SoftLayer/managers/account.py [hunk 2]
```diff
@@ -303,3 +303,25 @@ def get_network_message_delivery_accounts(self):
...
+                    softwareDescription[longDescription,name,manufacturer],
+                    subnet"""
+
+        return self.client['SoftLayer_Account'].getActiveVirtualLicenses(mask=_mask)
+
+    def get_active_account_licenses(self):
+        """Gets all active account licenses.
+
+                        :returns: Active account Licenses
+                        """
```
Comment (allmightyspiff) [inline SoftLayer/managers/account.py:324]:
```suggestion
        """Gets all active account licenses.

        :returns: Active account Licenses
        """
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Make sure to change the actual license keys to invalid/random ones.

File: SoftLayer/fixtures/SoftLayer_Account.py [hunk 3]
```diff
@@ -1100,3 +1100,110 @@
...
+            "virtualizationPlatform": 0,
+            "requiredUser": "administrator@vsphere.local"
+        }
+    }
+]
+
+getActiveVirtualLicenses = [{
+    "id": 12345,
+    "ipAddress": "192.168.23.78",
+    "key": "PLSK.06866259.0000",
```
Comment (ATGE) [inline SoftLayer/fixtures/SoftLayer_Account.py:1182]:
```suggestion
    "key": "TEST.01234567.0000",
```

File: SoftLayer/fixtures/SoftLayer_Account.py [hunk 4]
```diff
@@ -1100,3 +1100,110 @@
...
+        "version": "6.0",
+        "virtualLicense": 0,
+        "virtualizationPlatform": 0,
+        "requiredUser": "administrator@vsphere.local"
+    }
+},
+    {
+        "accountId": 123456,
+        "capacity": "4",
+        "key": "4122M-ABXC05-K829T-098HP-00QJM",
```
Comment (ATGE) [inline SoftLayer/fixtures/SoftLayer_Account.py:1144]:
```suggestion
        "key": "1MOCK-1TEST-KEY00-098HP-00QJM",
```

File: SoftLayer/fixtures/SoftLayer_Account.py [hunk 5]
```diff
@@ -1100,3 +1100,110 @@
         "smtpAccess": "1"
     }
 ]
+
+getActiveAccountLicenses = [{
+    "accountId": 123456,
+    "capacity": "4",
+    "key": "M02A5-6CJ8L-J8R9H-000R0-CDR70",
```
Comment (ATGE) [inline SoftLayer/fixtures/SoftLayer_Account.py:1107]:
```suggestion
    "key": "M0CK5-TEST5-KEY00-000R0-CDR70",
```
Also, I think just an item on the list would be enough.

Comment (caberos) [reply SoftLayer/fixtures/SoftLayer_Account.py:1107]: is better a randomize this key

**Review by ATGE — COMMENTED** [review]
### TESTING
-  $slcli account licenses
the command works, just please add a name for each table showed, Control panel licenses, and VMware Licenses

File: SoftLayer/CLI/account/licenses.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """return the control panel and VMWare licenses"""
+
+    manager = AccountManager(env.client)
+
+    panel_control = manager.get_active_virtual_licenses()
```
Comment (FernandoOjeda) [inline SoftLayer/CLI/account/licenses.py:18]:
```suggestion
control_panel = manager.get_active_virtual_licenses()
```

File: SoftLayer/CLI/account/licenses.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
+"""Show the all account licenses."""
+# :license: MIT, see LICENSE for more details.
+import click
+from SoftLayer import utils
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.account import AccountManager
```
Comment (FernandoOjeda) [inline SoftLayer/CLI/account/licenses.py:8]:
```suggestion
import SoftLayer
```

File: SoftLayer/CLI/account/licenses.py [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """return the control panel and VMWare licenses"""
+
+    manager = AccountManager(env.client)
```
Comment (FernandoOjeda) [inline SoftLayer/CLI/account/licenses.py:16]:
```suggestion
manager = SoftLayer.AccountManager(env.client)
```

**Review by FernandoOjeda — COMMENTED** [review]
Looks good, just a few comments.

**Review by allmightyspiff — APPROVED** [review]
looks good to me.

**Pull request merged**